scss/_patterns_navigation.scss

```diff
@@ -93,6 +93,18 @@ $spv-navigation-logo-bottom-position: 0.125rem; // 2px when 1rem is 16px
     }
   }
 
+  %navigation-slide {
+    @include vf-transition(transform);
+
+    height: calc(100vh - $navigation-logo-banner-height);
+    overflow-y: auto;
+    position: absolute;
+    top: $navigation-logo-banner-height;
+    transform: translateX(100vw);
+    width: 100%;
```

Comment:
I did this because the body tag on vanilla examples has margins. If I use `100vw` wouldn't look clean because the navigation will be overlapping the space created by the margin.

Reply:
If the margin get's in the way we could remove it from this example. We do it in some places by adding `margin: 0` on body.

Like this:
https://github.com/canonical/vanilla-framework/blob/d40b5d1e6d52466666ae4dbe0512c7a0f5bcad74/templates/docs/examples/layouts/sticky-footer.html#L4-L8